### ISSUE TITLE
hover effect for the menu bar

### ISSUE THREAD

**Issue by surajgupt01** [responsively-org/responsively-app#1332]
![Screenshot 2024-10-27 223757](https://github.com/user-attachments/assets/97010657-5d74-413c-9f66-11bb1b08777e)
# 🚀 Feature Request

**Comment by GSO0** [comment]
Hello @surajgupt01 , I'd like to work on this specific task. Could you assign it to me.

**Comment by GSO0** [comment]
Hello @surajgupt01 , may i start working on this issue?

**Comment by surajgupt01** [comment]
Yes you can but I am new in open source contributions facing error in this
repo to run locally.
Can you help me with it

**Comment by Timon-D3v** [comment]
@surajgupt01 are you working on this issue? I would like to do it otherwise.

**Comment by Samyak137** [comment]
I want to work on that will you please assign it to me.

**Comment by SriKrishnaPhanish** [comment]
I'd like to work in this issue, could you assign this issue to me.

**Comment by IDevSharma1** [comment]
I want to work on this issue will you assign it to me?

**Comment by YogeshK34** [comment]
This issue seems to be a good starting point as I am just starting out in open source. I would be glad to work on this issue, can you assign it to me?

**Comment by YogeshK34** [comment]
I am working on this issue

**Comment by swayamvasavada** [comment]
I will be working on this Issue.

**Comment by swayamvasavada** [comment]
@surajgupt01 Nice Suggestion! But this repository is not of Responsively website, This respository is of Resposively app.

Responsilvely is a tool which it provides to developer to build responsive wesbites. Issue you mentioned is and website of that showcase this app. You can post this issue in [Responsively website repo](https://github.com/responsively-org/website.git)
